src/test/java/uk/gov/hmcts/darts/casedocument/service/impl/GenerateCaseDocumentSingleCaseProcessorImplTest.java

```diff
@@ -137,6 +143,8 @@ void testGenerationOfCaseDocument() {
         assertThat(savedCaseDocument.getCreatedBy()).isEqualTo(user);
         assertThat(savedCaseDocument.getLastModifiedBy()).isEqualTo(user);
         assertThat(savedCaseDocument.isHidden()).isEqualTo(false);
+
+        verify(caseRepository).save(courtCaseCaptor.capture());
```

Comment:
Should validate content of the capture to make sure the correct fields are being set on the object

Reply:
tried but can't as the save method return is mocked so you are basically setting the values and comparing the same which makes it pointless.